src/data_sheets_schema/schema/D4D_Preprocessing.yaml

```diff
@@ -118,4 +136,101 @@ classes:
\ No newline at end of file
+        slot_uri: dcterms:description
+
+
+  ImputationProtocol:
+    is_a: DatasetProperty
+    description: >
+      Description of data imputation methodology, including techniques used
+      to handle missing values and rationale for chosen approaches.
+    exact_mappings:
+      - rai:dataImputationProtocol
+    attributes:
+      imputation_method:
+        description: >
+          Specific imputation technique used (mean, median, mode, forward fill,
+          backward fill, interpolation, model-based imputation, etc.).
+        range: string
+        multivalued: true
+      imputed_fields:
+        description: >
+          Fields or columns where imputation was applied.
+        range: string
+        multivalued: true
+      imputation_rationale:
+        description: >
+          Justification for the imputation approach chosen, including
+          assumptions made about missing data mechanisms.
+        range: string
+      imputation_validation:
+        description: >
+          Methods used to validate imputation quality (if any).
+        range: string
+
+
+  AnnotationAnalysis:
+    is_a: DatasetProperty
+    description: >
+      Analysis of annotation quality, inter-annotator agreement metrics,
+      and systematic patterns in annotation disagreements.
+    exact_mappings:
+      - rai:dataAnnotationAnalysis
+    attributes:
+      inter_annotator_agreement_score:
+        description: >
+          Measured agreement between annotators (e.g., Cohen's kappa value,
+          Fleiss' kappa, Krippendorff's alpha).
+        range: float
+      agreement_metric:
+        description: >
+          Type of agreement metric used (Cohen's kappa, Fleiss' kappa,
+          Krippendorff's alpha, percentage agreement, etc.).
+        range: string
+      analysis_method:
+        description: >
+          Methodology used to assess annotation quality and resolve disagreements.
+        range: string
+      disagreement_patterns:
+        description: >
+          Systematic patterns in annotator disagreements (e.g., by demographic
+          group, annotation difficulty, task type).
+        range: string
+        multivalued: true
+      annotation_quality_details:
+        description: >
+          Additional details on annotation quality assessment and findings.
+        range: string
+        multivalued: true
+
+
+  MachineAnnotationTools:
+    is_a: DatasetProperty
+    description: >
+      Automated or machine-learning-based annotation tools used in dataset creation,
+      including NLP pipelines, computer vision models, or other automated labeling systems.
+    exact_mappings:
+      - rai:machineAnnotationTools
+    attributes:
+      tool_name:
+        description: >
+          Name of the automated annotation tool (e.g., spaCy, NLTK, Stanford NER,
+          custom ML model, GPT-based annotation).
+        range: string
+        multivalued: true
+      tool_version:

```

Comment:
This could get messy because it doesn't provide a way to link a specific tool to a specific version (both `tool_name` and `tool_version` are independent lists). Since `tool_name` is just a string field anyway, maybe the documentation could specify including the version details there, and this slot could be modified or removed.